src/main/scala/com/fortysevendeg/android/sqlite/logging/LogWrapper.scala

```diff
@@ -14,7 +16,17 @@ trait LogWrapper {
 
   def e(message: String, maybeThrowable: Option[Throwable] = None)
 
-  def notImplemented[T](result: T): T
+  def notImplemented[T](result: T): T = {
+    val message = "Method not implemented" :: (stackTraces map (s => s" at ${methodName(s)}(${fileName(s)})")).toList
+    w(message.mkString("\n"))
+    result
+  }
+
+  def logOnError[T](f: => T, msg: String = "") =
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Call by name parameter should be in last position](https://www.codacy.com/app/47deg/scala-sqlite-droid/file/809264718/issues/source?bid=2442814&fileBranchId=2738989#l25)